pkg/api/mlflow/api/request/log.go

```diff
@@ -0,0 +1,47 @@
+package request
+
+// ParamPartialRequest is a partial request object for different requests.
+type ParamPartialRequest struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// TagPartialRequest is a partial request object for different requests.
+type TagPartialRequest struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// MetricPartialRequest is a partial request object for different requests.
+type MetricPartialRequest struct {
+	Key       string `json:"key"`
+	Value     any    `json:"value"`
+	Timestamp int64  `json:"timestamp"`
+	Step      int64  `json:"step"`
+}
+
+// LogParamRequest is a request object for `POST mlflow/log-parameter` endpoint.
```

Comment:
```suggestion
// LogParamRequest is a request object for `POST mlflow/runs/log-parameter` endpoint.
```

Same remark for all the other comments below ⬇️

Reply:
done.